edgedb-client/src/builder.rs

```diff
@@ -1138,11 +1140,37 @@ impl Builder {
                         _ => {}
                     }
                 }
+                ServerMessage::ParameterStatusSystemConfig(config) => {
+                    let output = config.output()
+                        .map_err(ProtocolEncodingError::with_source)?;
+                    let codec = output.build_codec()
+                        .map_err(ProtocolEncodingError::with_source)?;
+                    let data = codec.decode(config.data.as_ref())
+                        .map_err(ProtocolEncodingError::with_source)?;
+                    if let Value::Object { shape, fields } = data {
+                        for (el, field) in shape.elements.iter().zip(fields) {
+                            if el.name == "session_idle_timeout" {
```

Comment:
Probably more will added. So let's do match. And let's print `log::debug` with the parameters that we didn't match.